meinberlin/apps/contrib/static/js/ajax_modal.js

```diff
@@ -0,0 +1,34 @@
+/* globals $ */
+$(function () {
+  var $ajaxModals = $('[data-toggle="ajax-modal"]')
+  var modalHTML = (
+    '<div class="modal">' +
+      '<div class="modal-dialog modal-lg" role="document">' +
+        '<div class="modal-content">' +
+          '<div class="modal-header"><h2 class="modal-title u-first-heading"></h2><button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>' +
+          '<div class="modal-body"></div>' +
+      '  </div>' +
+      '</div>' +
+    '</div>'
+  )
+  var $backdrop = $('<div class="modal-backdrop" />')
+
+  $ajaxModals.on('click', function (e) {
```

Comment:
I would prefer to use event delegation here `$(document).on('click', '[data-toggle="ajax-modal"]', function() {})`